cmd/pluginator/internal/builtinplugin/builtinplugin.go

```diff
@@ -72,10 +72,7 @@ func ConvertToBuiltInPlugin() (retErr error) {
 		if strings.HasPrefix(l, "//go:generate") {
 			continue
 		}
-		if strings.HasPrefix(l, "//noinspection") {
```

Comment:
I didn't replace this with a `nolint:` exclusion, because I think this is questionable to begin with: it assumes that the annotation in question only appears on code we are going to exclude from the copy, i.e. the KustomizePlugin var. Better to put the annotation on the same line and exclude only that IMO.